test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1185,7 +1206,7 @@ describe('emitConversationEvent()', () => {
     });
 
     expectedEvent.added.push(conversationState);
-    handler.emitConversationEvent('added', conversationState, undefined);
+    handler.emitConversationEvent('added', conversationState, undefined as any);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    handler.emitConversationEvent('added', conversationState, undefined as unknown);
```